haystack/preprocessor/queryexpander.py

```diff
@@ -0,0 +1,90 @@
+from langdetect import detect
```

Comment:
Langdetect uses third-party api , which has limitations. Do we need to include it ?

Reply:
@tanmaylaud Which third-party API are you referring to?

Reply:
My bad , I confused this with another library.